packages/nimble-components/src/table-column/date-text/cell-view/index.ts

```diff
@@ -0,0 +1,57 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { template } from '../../text-base/cell-view/template';
+import type {
+    TableColumnDateTextCellRecord,
+    TableColumnDateTextColumnConfig
+} from '..';
+import { styles } from '../../text-base/cell-view/styles';
+import { TableColumnTextCellViewBase } from '../../text-base/cell-view';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-date-text-cell-view': TableColumnDateTextCellView;
+    }
+}
+
+/**
+ * A cell view for displaying date/time fields as text
+ */
+export class TableColumnDateTextCellView extends TableColumnTextCellViewBase<
+TableColumnDateTextCellRecord,
+TableColumnDateTextColumnConfig
+> {
+    private columnConfigChanged(): void {
+        this.placeholder = this.columnConfig.placeholder;
```

Comment:
@jattasNI There was strong discussion about removing the concept of placeholder values from the table: https://github.com/ni/nimble/issues/1364

As this and number text column start dev should we confirm that decision? How should that impact this PR?

Reply:
Has design/UX had any input regarding placeholders in the table? If not (and I suspect not), I can pivot to removing that from our table columns, as that would be a precursor to this PR.

Reply:
That is the open question cited in the linked issue that should be resolved

Reply:
I've created PR #1365 to remove placeholder support from table columns.